content/workshops/business-intelligence-tools/_index.md

```diff
@@ -1,8 +0,0 @@
----
-_db_id: 13
-content_type: workshop
-ready: true
-title: Business intelligence tools
----
-
```

Comment:
why was this deleted?

Reply:
There is definately information in the slides. 

Waere our data science peeps consulted? I think this is referenced in their syllabus. But they are pretty junior so I'm not sure that they should be culling parts of the syllabus anyway